go/felix/set/set.go

```diff
@@ -88,3 +89,15 @@ func (set mapSet) Copy() Set {
 	}
 	return cpy
 }
+
+func (set mapSet) Equals(other Set) bool {
```

Comment:
Unfortunately no, I don't think I can, because Equals is defined on the Set interface, not on mapSet.  We'd need another Set interface method to identify the underlying data that needs to be checked for equality.